language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionUtil.java

```diff
@@ -448,6 +451,24 @@ public static List<TextEdit> getTypeGuardCodeActionEdits(String varName, Range r
         return edits;
     }
 
+    public static List<TextEdit> addGettersCodeActionEdits(String varName, Range range, int offset, String typeName) {
+        Position startPos = range.getEnd();
+        Range newTextRange = new Range(startPos, startPos);
+        List<TextEdit> edits = new ArrayList<>();
+        String spaces = StringUtils.repeat(' ', offset);
+        edits.add(new TextEdit(newTextRange, generateGetterFunctionBodyText(varName, typeName, spaces)));
+        return edits;
+    }
+
+    public static List<TextEdit> addSettersCodeActionEdits(String varName, Range range, int offset, String typeName) {
+        Position startPos = range.getEnd();
+        Range newTextRange = new Range(startPos, startPos);
+        List<TextEdit> edits = new ArrayList<>();
+        String spaces = StringUtils.repeat(' ', offset);
+        edits.add(new TextEdit(newTextRange, generateSetterFunctionBodyText(varName, typeName, spaces)));
+        return edits;
+    }
```

Comment:
Shall we get the repeating code parts to a common place?

Reply:
Updated in 9b8ee2e211c6ab47082d00b3379ae13676eb610d